content/blog/2023-10-31-cis-benchmarks-update.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "CIS Benchmarks Update"
+type: blog
+author:
+ name: "Simon John"
+ bio: "Security Standards Architect"
+ image: /users/sjohn.png
+date: '2023-10-31'
```

Comment:
```suggestion
date: '2023-11-28'
```

Reply:
I realized next week will be 8.9 and then thanksgiving in the US, so we'll want to do this one the week after.

Reply:
you might want to rename the file eventually too